### DISCUSSION TITLE
array/prefix-sum-array

### DISCUSSION THREAD

**Discussion by giscus** [XuShaohua/algorithms-rs#2] [General]
# array/prefix-sum-array

https://algs.biofan.org/array/prefix-sum-array.html

**Comment by TangQingBi** [comment]
您好，计算 l..=r 范围和的公式，应当减去的是 left - 1。在下方的代码描述中，是正确的实现。

**Reply by XuShaohua** [reply]
已修复, 感谢提醒!
